nmdc_runtime/site/ops.py

```diff
@@ -1043,6 +1045,48 @@ def site_code_mapping() -> dict:
         )
 
 
+@op(
+    required_resource_keys={"mongo"},
+    config_schema={
+        "source_ontology": str,
+        "output_directory": Field(Noneable(str), default_value=None, is_required=False),
+        "generate_reports": Field(bool, default_value=True, is_required=False),
+    },
+)
+def load_ontology(context: OpExecutionContext):
+    cfg = context.op_config
+    source_ontology = cfg["source_ontology"]
+    output_directory = cfg.get("output_directory")
+    generate_reports = cfg.get("generate_reports", True)
```

Comment:
Curious as to what values are being supplied for these two parameters at runtime? Since we're using the dagster harness, presumably a folder path on the filesystem where the dagster infrastructure has been deployed?